### ISSUE TITLE
Custos renders off the staff

### ISSUE THREAD

**Issue by FGasper** [gregorio-project/gregorio#1473]
```
name: 1st Sunday of Advent ;
user-notes: OFFERTORY Ad te Domine;
commentary: Psalm 25 ;
annotation: ii ;
centering-scheme: english;
%fontsize: 12;
%spacing: vichi;
%font: OFLSortsMillGoudy;
%width: 5;
%height: 8.5;
%%
(f3)To(c) you,(ef/efef/HHhg) O(f) Lord,(F/GhvGFggf) (,) I(f) lift(fe/f//gwh) up(gfHHhvFEfHHh) my(f) soul;(fgf) (:)
```
^^ The above renders with gregorio 5.2.1 (LuaLaTeX Version 1.10.0  from TeX Live 2019/MacPorts 2019.50896_0) thusly, with the custos hanging off the end of the staff:
<img width="544" alt="image" src="https://user-images.githubusercontent.com/844867/62785285-58d53180-ba8e-11e9-9593-db102e27bc1a.png">

**Comment by Nokel81** [comment]
Having done some investigation myself, it seems to be that Gregorio is unwilling to spit up long numes. The solution that I have come up with is adding `z` (or break) characters in the middle of the numes.

**Comment by Nokel81** [comment]
A possible solution would be to have a character which means "you can break here" or have a set of rules of where gregorio will break a long nume.

**Comment by FGasper** [comment]
Is it possible to put the long neume itself at the start of a new line?

-F

**Comment by Nokel81** [comment]
Yes it is possible to manually do it by placing a `z` at the beginning of the nume.

`I(f) lift(fe/fgwh) up(zgfHHhvFEfHHh)`

**Comment by FGasper** [comment]
That would seem to defeat one of the benefits of Gregorio: that it (like TeX) separates content from presentation.

I’m hardly capable of making the change myself, and Gregorio is already such a fantastic tool that I hesitate to say, but would Gregorio be able to determine for itself when to break before a long neume?

-F

**Comment by rpspringuel** [comment]
Gregoriotex already does do some breaking.  So it ought to be just certain sequences that are causing the problems.  If you can identify the problematic sequence and where the appropriate breakpoint is in the sequence we can probably look into modifying the breaking logic.  

The idea of an optional break may also be possible, but I would need to look further at that to determine how it might be reasonably accomplished. 

✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️✝️
Fr. Samuel, OSB
(R. Padraic Springuel)
St. Anselm’s Abbey 
4501 South Dakota Ave, NE
Washington, DC, 20017
202-269-2300
(c) 202-853-7036

PAX ☧ ΧΡΙΣΤΟΣ

**Comment by Nokel81** [comment]
Given that gregorio does breaking on bar lines. We have at least the following sequences:

* `(f'ivHG'Fe!hhhvGFggf.)`
![image](https://user-images.githubusercontent.com/8225332/71643149-7b592880-2c83-11ea-9d3c-5edc59d25bf7.png)
* `(gfHHhvFEfHHh)`
![image](https://user-images.githubusercontent.com/8225332/71643155-975cca00-2c83-11ea-817c-a532e5931007.png)

That gregorio won't break up.

**Comment by rpspringuel** [comment]
Maybe #1184 again?

**Comment by eschwab** [comment]
Looks like it. Simply adding a `/` in places were a line break is acceptable seems to be a fix for me. This allows the score to be used flexibly with various widths without he need for cumbersome `z`'s.

**Comment by fiat96** [comment]
I agree with @eschwab's solution of adding one or more `/`'s into the score.

I believe that the rules that Gregorio seems to be following here are solid.  I'll elaborate:

`(gfHHhvFEfHHh)` is the unbreakable string.

`gf` forms a clivis, which is followed by two _puncta inclinata_, `HH`.  Allowing inclinata to automatically separate from the preceding neum would be potentially messy.

The inclinata are follwed by only a single non-inclined note `hv`.  Perhaps a line break could reasonably be inserted here in the given instance.  However, if Gregorio is trying to keep the inclinata `HH` attached to the single non-inclined note that follows them, I believe that is a good rule.  [I would not want a line break inserted into something like `hvGFg`.]

This pattern continues through the whole segment, with only one non-inclined note between groups of _inclinata_.  

I think Gregorio is making good decisions here.
